File: BackEnd/VB.NET/ServerSide.vb
Public Class ServerSide
    Function GetLocalFiles(Paramaters As String) As String
        'MsgBox("GetLocalFiles-Paramaters:" + Paramaters)
        Return "['test','test2']"
    End Function
    Function SomethingElse(Paramaters As String) As String
        'MsgBox("SomethingElse-Paramaters:" + Paramaters)
        Return "SomethingElse"
    End Function
End Class

File: BackEnd/CSharp/ServerSide.cs

namespace WebApp
{
    public class ServerSide
    {
        public string GetLocalFiles(string Paramaters)
        {
            // MsgBox("GetLocalFiles-Paramaters:" + Paramaters)
            return "['test','test2']";
        }

        public string SomethingElse(string Paramaters)
        {
            // MsgBox("SomethingElse-Paramaters:" + Paramaters)
            return "SomethingElse";
        }
    }
}